### PR TITLE
Update MQ Autogen Role Test As Non Blocking

### PR DESCRIPTION
**Pull request by GavinZZ**
### Description of changes
Update the integ test as non blocking
Addd it to trasnform test

### Description of how you validated changes
Transform test passed.

### Checklist

- [ ] Adheres to the [development guidelines](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#development-guidelines)
- [ ] Add/update [transform tests](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#unit-testing-with-multiple-python-versions)
    - [ ] Using correct values
    - [ ] Using wrong values
- [ ] Add/update [integration tests](https://github.com/aws/serverless-application-model/blob/develop/INTEGRATION_TESTS.md)

### Examples?

Please reach out in the comments if you want to add an example. Examples will be 
added to `sam init` through [aws/aws-sam-cli-app-templates](https://github.com/aws/aws-sam-cli-app-templates).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by aahung — APPROVED** [review]

File: integration/combination/test_function_with_mq.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def companion_stack_outputs(self, get_companion_stack_outputs):
             ),
         ]
     )
+    @nonblocking
```
Comment (hoffa) [inline integration/combination/test_function_with_mq.py:28]: Isn't it missing the `combination/function_with_mq` transform test?

File: .cfnlintrc.yaml [hunk 2]
```diff
@@ -122,6 +122,7 @@ ignore_templates:
   - tests/translator/output/**/function_with_runtime_config.json  # RuntimeManagementConfig
   - tests/translator/output/**/managed_policies_minimal.json  # Intentionally has non-existent managed policy name
   - tests/translator/output/**/function_with_mq.json  # Property "EventSourceArn" can Fn::GetAtt to a resource of types [AWS::DynamoDB::GlobalTable, AWS::DynamoDB::Table, AWS::Kinesis::Stream, AWS::Kinesis::StreamConsumer, AWS::SQS::Queue]
+  - tests/translator/output/**/function_with_mq_using_autogen_role.json  # Property "EventSourceArn" can Fn::GetAtt to a resource of types [AWS::DynamoDB::GlobalTable, AWS::DynamoDB::Table, AWS::Kinesis::Stream, AWS::Kinesis::StreamConsumer, AWS::SQS::Queue]
```
Comment (aahung) [inline .cfnlintrc.yaml:125]: til `Fn::GetAtt` is a verb 😂

File: integration/combination/test_function_with_mq.py (hunk 1 above)
Comment (GavinZZ) [reply integration/combination/test_function_with_mq.py:28]: The transform test was already added before. https://github.com/aws/serverless-application-model/pull/3045/files

File: .cfnlintrc.yaml (hunk 2 above)
Comment (GavinZZ) [reply .cfnlintrc.yaml:125]: lol

**Review by hoffa — APPROVED** [review]

**Review by xazhao — APPROVED** [review]

**Pull request merged**